src/sentry/static/sentry/app/views/projectDetail/projectDetail.tsx

```diff
@@ -122,17 +198,25 @@ class ProjectDetail extends AsyncView<Props, State> {
             <Layout.Body>
               <StyledSdkUpdatesAlert />
               <Layout.Main>
-                <ProjectScoreCards organization={organization} />
-                {[0, 1].map(id => (
-                  <ProjectCharts
-                    location={location}
-                    organization={organization}
-                    router={router}
-                    key={`project-charts-${id}`}
-                    index={id}
-                  />
-                ))}
-                <ProjectIssues organization={organization} location={location} />
+                <ProjectScoreCards
+                  organization={organization}
+                  isProjectStabilized={isProjectStabilized}
+                  selection={selection}
+                />
+                {isProjectStabilized && (
+                  <React.Fragment>
+                    {[0, 1].map(id => (
```

Comment:
Please why are we using an array of numbers here? why length 2?

Reply:
Right now we show two charts on the project detail page - we are experimenting with showing more/fewer charts there and this allows us to do so.